src/reuse/helper/errorMessages.ts

```diff
@@ -0,0 +1,21 @@
+export class ErrorMessages {
```

Comment:
we can discuss about making the functions non-static and add a property `functionName`. With that we only have to set the function name once during generation of the ErrorMessage object.

But not sure if its beneficial. Will discuss first

Reply:
will not do that